File: 백준/Bronze/2562. 최댓값/최댓값.py
nums = [int(input()) for _ in range(9)] 

max_value = nums[0]
index = 1

for j in range(1, 9):
    if nums[j] > max_value:
        max_value = nums[j]
        index = j + 1

print(max_value)
print(index)

File: 백준/Bronze/2562. 최댓값/최댓값.java


import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;


class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));




        int[] nums = new int[9];
        
        for (int i = 0; i < 9; i++) {
            nums[i] = Integer.parseInt(br.readLine());
        }
        
        int max = nums[0];
        int index = 1;
        
        for (int j = 1; j < 9; j++) {
            if (nums[j] > max) {
                max = nums[j];
                index = j + 1;
            }
        }
        System.out.println(max);
        System.out.println(index);


        }







    }



